packages/create-cloudflare/templates/next/c3.ts

```diff
@@ -31,29 +32,55 @@ const generate = async (ctx: C3Context) => {
 	updateStatus("Created wrangler.json file");
 };
 
-const updateNextConfig = (usesTs: boolean) => {
+const setupDevPlatformCode =
+	`import { setupDevPlatform } from '@cloudflare/next-on-pages/next-dev';
+
+	// Here we use the @cloudflare/next-on-pages next-dev module to allow us to
+	// use bindings during local development (when running the application with \`next dev\`).
+	// For more information see:
+	// https://github.com/cloudflare/next-on-pages/blob/main/internal-packages/next-dev/README.md
+	if (process.env.NODE_ENV === 'development') {
```

Comment:
@dario-piotrowicz do we need `(process.env.NODE_ENV === 'development')` here? (or could it be inside `setupDevPlatform` ?

(The code here is only moved from l42 so not directly related to the PR)

Reply:
This check is totally unnecessary, moving it inside `setupDevPlatform` wouldn't actually accomplish anything

it's here to try to make it extra clear to the developer that this function is only needed in development, in reality this only has any effect during development anyways

I'm happy to get rid of it if you want 👍

Reply:
> This check is totally unnecessary

Then I would drop it.

And in [next-on-pages](https://github.com/cloudflare/next-on-pages/blob/2cd4c3c704a00e6b693229f1f14102abc6318d11/pages-e2e/fixtures/issue797app/next.config.mjs#L6) as well